src/accessibilityPage/accessibilityPage.component.test.tsx

```diff
@@ -16,25 +22,50 @@ const dummyClasses = {
 
 describe('Accessibility page component', () => {
   let mount;
+  let mockStore;
   let props: CombinedAccessibiiltyPageProps;
+  let state: StateType;
 
   beforeEach(() => {
     mount = createMount();
-
+    mockStore = configureStore([thunk]);
     props = {
       classes: dummyClasses,
     };
+
+    state = {
+      scigateway: { ...initialState, authorisation: { ...authState } },
+      router: { location: createLocation('/') },
+    };
   });
 
   const theme = buildTheme(false);
 
   it('should render correctly', () => {
+    const testStore = mockStore(state);
+
     const wrapper = mount(
-      <MuiThemeProvider theme={theme}>
-        <AccessibiiltyPageWithStyles {...props} />
-      </MuiThemeProvider>
+      <Provider store={testStore}>
+        <MuiThemeProvider theme={theme}>
+          <AccessibiiltyPageWithStyles {...props} />
+        </MuiThemeProvider>
+      </Provider>
     );
 
     expect(wrapper.find('#accessibility-page')).toBeTruthy();
   });
+
+  it('should render contact us component correctly', () => {
+    const testStore = mockStore(state);
+
+    const wrapper = mount(
+      <Provider store={testStore}>
+        <MuiThemeProvider theme={theme}>
+          <AccessibiiltyPageWithStyles {...props} />
+        </MuiThemeProvider>
+      </Provider>
+    );
+
+    expect(wrapper.find('#contact-us')).toBeTruthy();
```

Comment:
Could we not just have this assertion in the previous test? Since we aren't setting up the state of the test differently

Reply:
I'll make the change